### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by v-atamanenko**

### CODE REVIEW

**Comment by v-atamanenko** [comment]
[UC React Widget Open-sourcing](https://trello.com/c/Fc0q9UN8/133-uc-react-widget-open-sourcing)

File: readme.md [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+
+<a href="https://uploadcare.com/?utm_source=github&utm_campaign=uploadcare-tinymce">
+  <img align="right" width="64" height="64"
+    src="https://ucarecdn.com/2f4864b7-ed0e-4411-965b-8148623aa680/uploadcare-logo-mark.svg"
+    alt="">
+</a>
+
+This is a React component made to easily integrate
+[Uploadcare Widget][uc-feature-widget] into your React app; convenient props
+management and lazy loading are bundled.
```
Comment (dayton1987) [inline readme.md]: to integrate Uploadcare Widget into your React app easily

File: readme.md [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
+
+The component allows users to upload media from their devices, social media,
```
Comment (dayton1987) [inline readme.md]: upload media > upload files

File: readme.md [hunk 3]
```diff
@@ -1,13 +1,40 @@
...
+cloud storage, and more. All that without any backend code that's usually
+required to handle uploads.
+
+**Note: This library comes untranspiled**. It means, if you want to support
+IE11, make sure you transpile `node_modules`. → [Read More][es6-debate] ←
```
Comment (dayton1987) [inline readme.md]: Is it really **THAT IMPORTANT**, couldn't we just stick with `Note, blabla, [learn more](<source>)`?

File: readme.md [hunk 4]
```diff
@@ -1,13 +1,40 @@
...
+* [Feedback](#feedback)
 
-**This library comes untranspiled**
+## Demo and Examples
 
-Want to support IE11 → make sure you transpile node_modules.
+- [Sandbox Demo](https://codesandbox.io/s/uploadcarereact-widget-7xpqp)
+- [Props](https://codesandbox.io/s/uploadcarereact-widget-props-example-oqk0v)
+- [onChange](https://codesandbox.io/s/uploadcarereact-widget-onchange-example-o376j)
+- [Gatsby](https://codesandbox.io/s/gatsby-starter-default-jr6nq)
```
Comment (dayton1987) [inline readme.md]: pls briefly elaborate on the purpose of each demo

File: readme.md [hunk 5]
```diff
@@ -23,31 +50,34 @@ import { Widget } from "@uploadcare/react-widget";
 <Widget publicKey="demopublickey" />;
```
Comment (dayton1987) [inline readme.md]: what is that key and where it comes from?

File: readme.md [hunk 6]
```diff
@@ -23,31 +50,34 @@ import { Widget } from "@uploadcare/react-widget";
...
+## Configuration
 
-All the props except for the listed below are passed down to the widget. See [widget documentation](https://uploadcare.com/docs/file_uploads/widget/options/#options) for the details.
+### Component configuration
 
-### `value`: `string`
+#### `value`: `string`
 
-Set a file/group UUID or a [CDN link](https://uploadcare.com/docs/delivery/) as a value.
+Set a file/group UUID or a [CDN URL][delivery-docs] as a value.
```
Comment (dayton1987) [inline readme.md]: So I already know what file/group UUIDs are, but am not yet aware of the CDN URLs?
pls add the reference to `[file/group UUID](<source>)`

File: readme.md [hunk 7]
```diff
@@ -23,31 +50,34 @@ import { Widget } from "@uploadcare/react-widget";
...
 
-### `onFileSelect`: `(fileInfo: FileInfo) => void`
+#### `onFileSelect`: `(fileInfo: FileInfo) => void`
 
-### `onChange`: `(fileInfo: FileInfo) => void`
+#### `onChange`: `(fileInfo: FileInfo) => void`
 
-`FileInfo` object is described [here](https://uploadcare.com/docs/api_reference/rest/accessing_files/#properties)
+Set callbacks for the respective events. The `FileInfo` object is 
+described [here][api-refs-props].
```
Comment (dayton1987) [inline readme.md]: Set callbacks ...:

* onFileSelect
* onChange

?

File: readme.md [hunk 8]
```diff
@@ -23,31 +50,34 @@ import { Widget } from "@uploadcare/react-widget";
...
 
-### `customTabs`: `{[string]: CustomTabConstructor}`
+#### `customTabs`: `{[string]: CustomTabConstructor}`
 
-Set a [custom tabs](https://uploadcare.com/docs/api_reference/javascript/custom_tabs/) for a widget.
+Set [custom tabs][custom-tabs-docs] for a widget.
```
Comment (dayton1987) [inline readme.md]: I'd say we add custom tabs rather than set them. Also, no notes on whether I need those at all.

File: readme.md [hunk 9]
```diff
@@ -57,16 +87,49 @@ function myTab(container, button, dialogApi, settings, name, uploadcare) {
...
+
+## Widget configuration
+
+Uploadcare Widget can be deeply customized to suit your UX/UI. You can define
+allowed upload sources, implement file validation, and more.
+
+All the options defined in the [widget options docs][widget-options-docs] are
+also supported in the component as props.
+
+Use our live [widget sandbox][uc-widget-configure] as a starting point and consider
```
Comment (dayton1987) [inline readme.md]: use the live Uploadcare Widget Configurator

File: readme.md [hunk 10]
```diff
@@ -57,16 +87,49 @@ function myTab(container, button, dialogApi, settings, name, uploadcare) {
...
+checking out the docs on [widget configuration][uc-docs-widget-config] and its
+[JavaScript API][uc-docs-widget-js-api].
```
Comment (dayton1987) [inline readme.md]: do we really need to mention the js api here?

**Pull request merged**